classes/class-block-details-list.php

```diff
@@ -108,34 +118,40 @@ public function column_post_title( $item ) {
 	/**
 	 * Get the table data
 	 *
-	 * @return mixed
+	 * @param  array  $per_page  Number of posts to be retrieved.
+	 * @param  string $paged     Current page number.
+	 *
+	 * @return array
 	 */
 	private function get_block_details( $per_page = 10, $paged = 1 ) {
-		$block = $_GET['block'];
-		$block_name_srtlen = strlen( $block );
 		$data = array();
-		
-		if( $block ) {
+
+		if ( isset( $_GET['block'] ) ) { // phpcs:ignore
```

Comment:
It's better to ignore one or more specific rules for this line than all of them. And this change should be applied to all `// phpcs:ignore`